test_conformance/vulkan/vulkan_interop_common/vulkan_list_map.cpp

```diff
@@ -16,6 +16,7 @@
 
 #ifdef _WIN32
 #define NOMINMAX
+#include <algorithm>
```

Comment:
Can this be moved out of the `_WIN32` block?  It shouldn't be Windows-specific.

Note also that we already define `NOMINMAX` globally via CMake options, so we shouldn't need to define it explicitly here.  This means that the entire `_WIN32` block can be removed, and we can just add the include for `<algorithm>`.

Reply:
fixed